### PR TITLE
Add second article for reproducible builds

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

**Review by marmarek — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA256

Technically looks good, may deserve another pass by @andrewdavidwong on the language side
Approved as of 6217b0739d536a73a2631112434b276d4feb9ae7
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCAAdFiEEnYWYU8Ll1fIX9iss8yiUvpaEk4oFAmFgbCYACgkQ8yiUvpaE
k4ruAw//Vb6ukkMZfRlledpKHptkPtJl3ASW9pghw3iDnlgP+dbkceHLzi7Z1aI+
FlSCyXWui9Bn0AJj2jARmsUXt9oS9z73cMSN/GcrWxBIUHRHWp+0mUAIAsSJrsUv
omdjj0SdQUu7kjxCxs5noTA3pFiSZPzyt9uWXq+qj+mGy7jIPjFSY5KV7V0Ys7op
Yyd/Q+5nyl7rcBcYcpS9z5w+b2tX+EZ83V4dEroSb4i8kfhOyXpBaVBEK2Tcy89P
gJ178It2v9d6d72wAKbULONUHJ/uhxumurxLXmz6FRMbgifoml611RPADVfxmzXm
Ydsl604krLjxrwKP81CxbFnh7eB3rMgm9mQxM1pnEUOIN425NowJzfsOqk+0czky
UpfuHqZprrCGu3X7Kp+WaQ30EPu9etBtwllN0RTwEy8VhGP8PXwbR43XZGBI31g/
GQwdwjBMFcYe1EDOfYvgJCv+p+XEcDVL2mkW+yiTeThMxXGmgfLX2aOMS8aYp+PE
ae9Rsq7YhgrldSsJyh9WrAqAA9hqTDeHTLea3A/rqYvnhmJC7MlzDhNME53+x950
r1XvqVkWt5ViQ8ULywO1DDiju6DRi5HJOdyKryYRzVuxas2Beeo8Vsia0Z65E58t
SNsko1bFMOCpJ9A6yuJUHnAccorgmdniWDb8ZU4zjsAND3atqew=
=GT4U
-----END PGP SIGNATURE-----
```

**Comment by andrewdavidwong** [comment]
@fepitre, thanks for this great article, and congratulations on all you've accomplished! This is a great work!

I've copyedited the text while attempting to preserve your original style and voice. Please have a look to make sure you're happy with the results before publishing.

File: 2021-10-08-reproducible-builds-for-debian-a-big-step-forward.md [hunk 1]
```diff
@@ -1,42 +1,42 @@
...
...een made. Then, we developed `resume` and `retry` download functions, which unfortunately are brute force download functions. For storing the data, a simple approach has been employed: storing files as SHA-256 names, then creating symlinks to reconstruct the repository layout. In order to get file information (package and repository metadata), we rely on simply reading a symlink. It took 3-4 months to get 4.2 TB of data, representing just 2017 (at the time of writing). Most of the information about the downloaded files and their source repository is stored in a database. In parallel, we added --- like the original `snapshot.debian.org` --- an API, [snapshot-api](https://github.com/fepitre/debian-snapshot#API), to expose information about repositories. Unlike the original one, we added much more information that rebuilder software, e.g. `debrebuild`, needs to have when requesting package information, such as the exact location of a given package in terms of Debian archive, timestamp, suite, architecture and component. The service is now publicly exposed at <https://snapshot.notset.fr> and the API endpoints at <https://snapshot.notset.fr/mr>. The service is home-hosted by the author.
```
Comment (fepitre) [inline 2021-10-08-reproducible-builds-for-debian-a-big-step-forward.md:24]: The 4.2 TB of data represent the whole time window from 2017 to today, not just 2017 (hopefully..)

File: 2021-10-08-reproducible-builds-for-debian-a-big-step-forward.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ Is rebuilding Debian really possible?
...
...pshot had been made. Then, we developed `resume` and `retry` download functions, which unfortunately are brute force download functions. For storing the data, a simple approach has been employed: storing files as SHA-256 names, then creating symlinks to reconstruct the repository layout. In order to get file information (package and repository metadata), we rely on simply reading a symlink. It took 3-4 months to get 4.2 TB of data, which represents 2017 to the present. Most of the information about the downloaded files and their source repository is stored in a database. In parallel, we added --- like the original `snapshot.debian.org` --- an API, [snapshot-api](https://github.com/fepitre/debian-snapshot#API), to expose information about repositories. Unlike the original one, we added much more information that rebuilder software, e.g. `debrebuild`, needs to have when requesting package information, such as the exact location of a given package in terms of Debian archive, timestamp, suite, architecture and component. The service is now publicly exposed at <https://snapshot.notset.fr> and the API endpoints at <https://snapshot.notset.fr/mr>. The service is home-hosted by the author.
```
Comment (andrewdavidwong) [inline 2021-10-08-reproducible-builds-for-debian-a-big-step-forward.md:24]: @fepitre, how's this?

**Pull request merged**